spyder/plugins/editor/lsp/client.py

```diff
@@ -57,6 +57,8 @@ class LSPClient(QObject, LSPMethodProviderMixIn):
     # Signals
     sig_initialize = Signal(dict, str)
 
+    sig_lsp_error = Signal(str)
```

Comment:
Please change this to `sig_server_error` and add the following comment above:

    # This signal is used to report internal server errors through
    # Spyder's facilities.